portfolio/src/main/webapp/script.js

```diff
@@ -63,24 +63,21 @@ async function sendAndGet() {
   });
 }
 
-function getCommentsHome() {
-  fetch('/data').then(response => response.json()).then((msgs) => {
-    const statsListElement = document.getElementById('home-comments-container');
-    statsListElement.innerHTML = '';
-    msgs.forEach((msg) => {
-      statsListElement.appendChild(
-        createListElement(msg.nickname + ': ' + msg.message));
-      statsListElement.appendChild(
-        createImgElement(msg.image));
-    })
-  });  
-}
-
+// Makes a Get request and loads Posts to the Blog page on body load
+// Also loads 5 display-only posts to be displayed on the home page.
 async function loadPosts(){
   fetch('/data').then(response => response.json()).then((msgs) => {
     const statsListElement = document.getElementById('posts-list');
+    const homeListElement = document.getElementById('home-comments-container');
+    var i = 1;
     msgs.forEach((msg) => {
       statsListElement.appendChild(createListElement(msg));
+      console.log('value :' + i);
+      if (i > 5) {
+        return true;
```

Comment:
What does this function return? I ask because this is the only place it returns a value (true).

Reply:
It doesn't return anything. I needed something like "continue", but in JQuery (in this case the forEach method) continue doesn't work. I did a quick search and it looks like returning true is how people do continue in jQuery.

https://stackoverflow.com/questions/7028716/illegal-continue-statement-in-javascript-each-loop